google/cloud/internal/random.h

```diff
@@ -36,28 +36,21 @@ std::vector<unsigned int> FetchEntropy(std::size_t desired_bits);
 // in particular.
 using DefaultPRNG = std::mt19937_64;
 
-/**
- * Initialize any of the C++11 PRNGs in `<random>` using std::random_device.
- */
-template <typename Generator>
-Generator MakePRNG() {
-  // We need to get enough entropy to fully initialize the PRNG, that depends
-  // on the size of its state.  The size in bits is `Generator::state_size`.
-  // We convert that to the number of `unsigned int` values; as this is what
-  // std::random_device returns.
-  auto const desired_bits = Generator::state_size * Generator::word_size;
-
+/// Create a new PRNG.
+inline DefaultPRNG MakeDefaultPRNG() {
+  // Fetch a few (typically 64) bits of entropy to initialize the PRNG. We used
+  // to fetch a lot more entropy, but that is overkill for our purposes.
```

Comment:
I'd reword to avoid talking about what we used to do, and rather speak to what we now do.

Perhaps just _"Fetch a few bits ..., which is sufficient for our purposes"_?